com.ibm.wala.core/src/main/java/com/ibm/wala/ipa/callgraph/propagation/cfa/AllocationString.java

```diff
@@ -0,0 +1,52 @@
+package com.ibm.wala.ipa.callgraph.propagation.cfa;
+
+import com.ibm.wala.ipa.callgraph.ContextItem;
+import com.ibm.wala.ipa.callgraph.propagation.AllocationSite;
+import java.util.Arrays;
+
+/** @author genli */
```

Comment:
Please add a proper Javadoc comment explaining what this class represents

Reply:
@msridhar  I think this is mainly due to the excellent design of wala :+1:.  I have added the javadoc and copyright header